Makefile

```diff
@@ -14,7 +14,6 @@ else
   endif
   HOST_ARCH := amd64
 endif
-undefine HOST_HARDWARE
```

Comment:
That has been introduced in 3.82 :smirk:

https://git.savannah.gnu.org/cgit/make.git/tree/NEWS?id=95f1a32d27eef91a8a037522467d7060e7da2624#n103